.pre-commit-config.yaml

```diff
@@ -37,6 +37,7 @@ repos:
                 notebooks/api_examples/.*.ipynb|
                 notebooks/benchmarks/.*.ipynb|
                 notebooks/genomic_examples/.*.ipynb|
+                notebooks/tabular_examples/tree_based_models/[\w\s]+.ipynb|
```

Comment:
What's the reason for using `[\w\s]+` instead of `.*` here?

Reply:
This is to exclude the `tree_based_models/tree_shap_paper` directory, which would otherwise be included in the pattern.